deployment-detector/src/agent.spec.ts

```diff
@@ -1,21 +1,27 @@
-import {
-  FindingType,
-  FindingSeverity,
-  Finding,
-  HandleTransaction,
-  TransactionEvent,
-  ethers,
-} from "forta-agent";
-import { provideTransactionHandler } from "./agent";
-import { botsParams, createFinding, FORTA_CREATE_AGENT, inputType, NETHERMIND_DEPLOYER } from "./utils";
+import { Finding, HandleTransaction, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+import { provideHandleTransaction } from "./agent";
+import { FORTA_AGENT_EVENT, FORTA_AGENT_REGISTRY } from "./utils";
 import { TestTransactionEvent } from "forta-agent-tools/lib/test";
 import { createAddress } from "forta-agent-tools";
 
-const randomAddress = createAddress("0x00");
+export type inputType = {
+  proxyAddress: string;
+  deployerAddress: string;
+  createEventAgent: string;
+};
 
+const otherDeployerAddress = createAddress("0x01");
+const mockOtherFortaContract: string = "0xB50d3960a49120D0A6B543E7295cAE6C78d07967";
+const mockDeployerAddress: string = "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
+const mockFortaAgentRegistry: string = "0x61447385B019187daa48e91c55c02AF1F1f3F863";
```

Comment:
It's a good practice to use addresses that are clearly mocks for testing, like you did for `otherDeployerAddress`.

The other could be `createAddress("0x02")`, etc.